internal/controllers/machinedeployment/machinedeployment_controller_test.go

```diff
@@ -343,38 +431,10 @@ func TestMachineDeploymentReconciler(t *testing.T) {
 			return len(machineSets.Items)
 		}, timeout*3).Should(BeEquivalentTo(1))
```

Comment:
Can we change the var in l.398 to newestMachineSet?

Because of in-place propagation we don't get a third MachineSet anymore, so the name is highly misleading